src/platformdirs/api.py

```diff
@@ -51,7 +52,10 @@ def __init__(
         self.multipath = multipath
         """
         An optional parameter only applicable to Unix/Linux which indicates that the entire list of data dirs should be
-        returned. By default, the first item would only be returned.
+        returned. By default, the first item would only be returned. This parameter does not affect the
+        :class:`pathlib.Path <pathlib.Path>`-returning methods, e.g. :meth:`platformdirs.site_data_path`. These always
+        return the first item. See `platformdirs/platformdirs#24
+        <https://github.com/platformdirs/platformdirs/issues/24>`_ for a discussion.
```

Comment:
This should not be changed. What you're discussing is related to the path variants, and even within that just some of them, and should be discussed there not here. Also no need to link the issue. Finally, the point here only applies to XDG and UNIX, so likely the best place to document it is also there.

Reply:
What do you think about moving the multipath-specific handling to `src/platformdirs/unix.py` then (i.e. overwrite the properties)? The ABC would keep its general implementation and assume that `site_*_dir` only returns one path.

Reply:
Yeah, that's alright too 👍 we can override the site properties in UNIX to handle the multipath case map to one 👍

Reply:
Resolved in 3af7fca